solidity/contracts/token/libs/TokenRouter.sol

```diff
@@ -130,7 +132,19 @@ abstract contract TokenRouter is GasRouter {
             _hook
         );
 
-        emit SentTransferRemote(_destination, _recipient, _amountOrId);
+        emit SentTransferRemote(_destination, _recipient, outboundAmount);
+    }
+
+    function _outbound(
```

Comment:
Definitely would appreciate some documentation and comments here

Reply:
As well as indicating the desired use case (which i believe is only scaling, right?

Reply:
[2c96dd2](https://github.com/hyperlane-xyz/hyperlane-monorepo/pull/4729/commits/2c96dd2e2025c849c461326b499e6726e492ccad)